server/apiserver/apiserver.go

```diff
@@ -220,19 +219,19 @@ func (api *ApiServer) HTTPUpdateUserList(w http.ResponseWriter, req *http.Reques
  */
 func (api *ApiServer) CallUpdateUserList(user_id string) []byte {
 	byt := []byte("")
-	_, ifErr := api.GetLastUserModification(user_id)
+	lastMove, ifErr := api.GetLastUserModification(user_id)
 	if (ifErr) {
 		err := api.SetLastUserModification(user_id, "0")
 		if (err) {
 			// Error from SetLastUserModification call
-			byt = makeCreateUserMessage(403)
+			byt = makeCreateUserMessage(403, -1)
 		} else {
 			// Successfully created the user
-			byt = makeCreateUserMessage(200)
+			byt = makeCreateUserMessage(200, 300000)
 		}
 	} else {
 		// User already existed
-		byt = makeCreateUserMessage(401)
+		byt = makeCreateUserMessage(401, math.Max(300000 - (time.now() - strconv.Atoi(lastMove)), 0))
```

Comment:
nit, but maybe you can make the 30,000 a variable before all of this logic? I've been meaning to bring up the idea of changing this to a much smaller time (like 15-30 seconds because 5 mins is way too long for the demo) and it'd be better if we only had to change it in one place here since we'll already have to change it on the client too